src/components/Loader.scss

```diff
@@ -0,0 +1,211 @@
+  left: 0;
+  border-radius: 50%;
+  animation: jelloShadowAnimation 500ms linear infinite;
+}
+
+@keyframes jelloBoxAnimation {
+  17% {
+    border-bottom-right-radius: $border-radius-small;
+  }
+  25% {
+    transform: translateY($su-small) rotate(22.5deg);
+  }
+  50% {
+    border-bottom-right-radius: $su-xlarge;
+    transform: translateY($su-default) scale(1,.9) rotate(45deg) ;
+  }
+  75% {
+    transform: translateY($su-small) rotate(67.5deg);
+  }
+  100% {
+    transform: translateY(0) rotate(90deg);
+  }
+}
+
+@keyframes jelloShadowAnimation {
+  50% {
+    transform: scale(1.2,1);
+  }
+}
+
+// * 1.b Pulse Indicator
+
+.PulseAnimation {}
+
+.PulseAnimation-dot {
+  display: inline-block;
+  width: $su-small;
+  height: $su-small;
+  background-color: $cu-info--light;
+  border-radius: 100%;
+  animation: PulseAnimation 1.5s infinite ease-in-out both;
+}
+
+.PulseAnimation-dot:nth-child(1) {
+  animation-delay: -500ms;
+}
+
+.PulseAnimation-dot:nth-child(2) {
+  animation-delay: -250ms;
+}
+
+@keyframes PulseAnimation {
+  0%, 80%, 100% {
+    transform: scale(0)
+  }
+  40% {
+    transform: scale(1.0)
+  }
+}
+
+// * 1.c Spin Indicator
+
+.SpinAnimation {
+  display: inline-block;
+  width: $su-large;
+  height: $su-large;
+  border-radius: 50%;
+  vertical-align: middle;
+  margin: $su-xsmall;
+  border-top: solid $su-xxsmall $cu-info;
+  border-right: solid $su-xxsmall transparent;
+  box-shadow: 0 0 0 1px $cu-info--light;
+  animation: SpinAnimation 1s linear infinite;
+}
+
+@keyframes SpinAnimation {
+  0% {
+    transform: rotate(0deg);
+  }
+  50% {
+    transform: rotate(180deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+// * 2 Page Loader
+
+.PageLoader {
+  width: 100%;
+  height: 125px;
+  text-align: center;
+  display: block;
+}
+
+.PageLoader-indicator {
+  position: relative;
+  display: inline-block;
+  height: $su-xlarge;
+  padding-left: $su-xlarge;
+  margin-bottom: $su-small;
+}
+
+.PageLoader-indicatorAnimation {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  margin: auto;
+}
+
+.PageLoader-indicatorText {
+  font-size: $tu-large-fontSize;
+  font-weight: $tu-bold-fontWeight;
+  line-height: $su-xlarge;
+  color: $cu-info;
+}
+
+.PageLoader-message {
+  display: block;
+  font-size: $tu-large-fontSize;
+  padding: $su-default 0;
+  color: $cu-info--light;
+}
+
+// * 2.a Page Loader Modifiers
+
+.PageLoader--centered {
+  position: fixed;
+  top: 50%;
+  left: 0;
+  margin-top: -63px;
+}
+
+.PageLoader--jello {
+
+  .PageLoader-indicator {
+    height: scaleGrid(6);
+    padding-left: scaleGrid(6);
+  }
+
+  .PageLoader-indicatorText {
+    line-height: scaleGrid(6);
+    color: $cu-info--light;
+    font-weight: $tu-base-fontWeight;
+  }
+
+  .PageLoader-message {
+    font-size: $tu-xlarge-fontSize;
+    font-weight: $tu-semibold-fontWeight;
+  }
+
+}
+
+// * 3 Block Loader
+
+.BlockLoader {
+  display: block;
+  text-align: center;
+  padding: $su-small 0;
+}
```

Comment:
Yeah I guess we could probably simplify both loaders down to a single loader, that just has elements that can be added on